app/assets/stylesheets/application.css.scss

```diff
@@ -124,16 +132,33 @@ input {
   border-color: white;
   border: 1px;
   border-radius: 4px;
+  height: 75%;
+  margin-top: 2%;
+  margin-right: 2%;
+  margin: 0 auto;
 }
 
 #results:empty {
   display: none;
 }
 
-.image-container > img {
+#content > img {
   // max-height: 200px;
-  height: 100%;
-  width: 100%;
+  width: 80%;
+  margin: 0 auto;
+  margin-top: 4%;
+  border-radius: 4px;
+  border: 1px;
+  border-style: solid; 
+  border-color: #9ECAE1;
+}
+
+.centerfy img{
+  margin: 0 auto;
+}
+
+.centerfy{
```

Comment:
Merge rule `.centerfy` with rule on line 156<br>Opening curly brace `{` should be preceded by one space